nym-wallet/src/components/Delegation/DelegateModal.tsx

```diff
@@ -171,6 +169,29 @@ export const DelegateModal: React.FC<{
     validate();
   }, [amount, identityKey]);
 
+  const resolveMixId = useCallback(
+    debounce(async (idKey) => {
+      if (!idKey || !validateKey(idKey, 32)) {
+        return;
+      }
+      let res;
+      try {
+        res = await tryConvertIdentityToMixId(idKey);
+      } catch (e) {
+        Console.warn(`failed to resolve mix_id for "${idKey}": ${e}`);
```

Comment:
Well, this can actually mean one of 2 things:
- we failed to resolve it (as you mention), implying some issue with the validator
- [more likely] mixnode with the specified identity key doesn't actually exist

Reply:
Yes, in both cases "failed to resolve mix_id" is generic enough, isn't it?
If not, what do you suggest?

Reply:
I'm not 100% sure how the message will get displayed to the user, so maybe that's fine after all